apps/uv/uv.star

```diff
@@ -31,37 +34,26 @@ uv_colors = [
 ]
 
 def main(config):
-    weather_data = get_weather_data(config)
-
-    if weather_data == None:
-        return []
+    if config.get("api_key") == None:
+        return render.Root(render.WrappedText("Fix config:\nMissing\nAPI key"))
```

Comment:
@matslina  Great idea - is there best practice for detecting when the app is in the demo screen, vs being added but not configured?

Reply:
It'd basically be this case. Possibly also when the field is the empty string `""`, so that the user gets the error message when they modify the field (unless they've entered a valid key).

Reply:
https://github.com/tidbyt/community/pull/1678